src/gt4py/backend/dace_backend.py

```diff
@@ -183,7 +183,7 @@ class DaCeComputationCodegen:
                 const int __J = domain[1];
                 const int __K = domain[2];
                 ${name}_t dace_handle;
-                auto allocator = gt::sid::make_cached_allocator(&${allocator}<char[]>);
+                auto allocator = gt::sid::cached_allocator(&${allocator}<char[]>);
```

Comment:
Is this just updated API or a semantic change?

Reply:
updated API